Gamemode Mods/Stable/Starcore_Pointslist/Data/Scripts/Additions/PointAdditions.cs

```diff
@@ -557,9 +557,9 @@ internal class PointAdditions : MySessionComponentBase
             ["Cat_StarcoreUlitity"] = 21,
             ["LargeBlockRemoteControl"] = 100,
             ["SmallBlockRemoteControl"] = 50,
-            ["LargeProgrammableBlock"] = 100,
-            ["LargeProgrammableBlockReskin"] = 100,
-            ["6SidePB"] = 100,
+            ["LargeProgrammableBlock"] = 50,
+            ["LargeProgrammableBlockReskin"] = 50,
+            ["6SidePB"] = 50,
             ["SmallProgrammableBlock"] = 50,

```

Comment:
No Smallgrids? !https://imgur.com/a/hEMMxeR

Reply:
i probably shouldn't use the total smallgrid death copypasta here, buuuuutttttt